### ISSUE TITLE
FlyleafHost not align in second time play

### ISSUE THREAD

**Issue by MinChanSike** [SuRGeoNix/Flyleaf#377]
FlyleafHost is not align when I'm trying to play in second time after stop. ( `Player.Stop();` ,  no Dispose)
I'm using **FlyleafLib.Controls.WinUI 1.0.18** in WinUI application.
For your reference, if you get some time to check please see my sample repo [here](https://github.com/MinChanSike/TestPlayerWinUIApp) .

![image](https://github.com/SuRGeoNix/Flyleaf/assets/2868221/baa9b6c3-0edb-4a72-8097-b8eff302183f)

**Comment by MinChanSike** [comment]
Sorry to make you confuse, 
Yes you are correct, the issue is video does not resize the form to fit.
It can be fit again if we resize the form, please see my screenshot below.

Thanks for information for player no need to stop to open new stream, the issue is because of playing after stop.
I may need to use stop function as a usecase, the library have some function to invalidate/refresh video size after stop?

https://github.com/SuRGeoNix/Flyleaf/assets/2868221/c4d6c68f-003e-45b5-9bc0-2330bc4b9c50

**Comment by SuRGeoNix** [comment]
@MinChanSike Can you help me reproduce it? It seems that this issue it only happens with the rtsp protocols? How do you create the local rtsp so I can test it?

**Comment by MinChanSike** [comment]
@SuRGeoNix According by my testing, unfortunately the issue is happen on `RTSP/RTMP/HLS/MPEG-DASH` protocols.
To create local rtsp/rtmp, I'm using **WowzaStreamingEngine** and pushing stream with ffmpeg.
```
ffmpeg.exe -re -stream_loop -1 -i sample.mp4 -c:v copy -c:a copy -f rtsp rtsp://127.0.0.1:1935/live/sample1
```

**Comment by SuRGeoNix** [comment]
@MinChanSike From your first screenshot, my understanding is that for WPF works fine? This sound for me more like an FFmpeg issue with the specify proto/format. I will give another try to reproduce it.

**Comment by MinChanSike** [comment]
yes, it is work fine on WPF FlyleafPlayer, only on WinUI got that issue.

**Comment by SuRGeoNix** [comment]
@MinChanSike I've used mediamtx and the same command as you suggest above to publish rtsp stream. I was not able to reproduce the issue. Can you provide the specified media file to test? (happens with all the files?)

**Comment by SuRGeoNix** [comment]
Yes I've used your repo. Can you provide the sample file that you use? I cannot reproduce it. Can you test it also with mediamtx?

**Comment by MinChanSike** [comment]
Yes, I also test with mediamtx and got same result.
https://github.com/SuRGeoNix/Flyleaf/assets/2868221/997bf9ef-8535-4d70-bb6b-af808df21a6a


Please take a look my test video file from google drive lik.
https://drive.google.com/file/d/13IP4iu2xom7nTRVQRYCLUohn6Qc4P5V-

This is my rtsp stream push command to mediamtx server.
```bash
ffmpeg.exe -re -stream_loop -1 -i sample_rb.mp4 -c:v copy -c:a copy -tune zerolatency -preset ultrafast -f rtsp rtsp://127.0.0.1:8554/live/sample1
```

Thank you @SuRGeoNix

**Comment by SuRGeoNix** [comment]
@MinChanSike I cannot reproduce it with Win10, I use latest MediaMTX v1.3.1 and tried different version of FFmpeg (4/5/6/master). So I suspect it's Win11 which causing this issue. I don't have a spare Win11 system for testing currently.

I think the easiest way for you currently fix this issue is to use Player.renderer.ResizeBuffers (not sure when exactly but after that it changes your control's width/height).

**Comment by MinChanSike** [comment]
@SuRGeoNix Thanks for your time and effort.
According by my testing, the issue is not because of source stream.
Even MediaMTX v1.3.1 and camera direct rtsp url, still got same issue.

I think it is related with #328, after changing my **Display Scaling** in Window Settings **125%** to **100%**  does not see this issue.
Also can resolve with `Player.renderer.ResizeBuffers`, but I may need to know previous FlyleafHost element size to correct back.

**Comment by SuRGeoNix** [comment]
@MinChanSike Nice finding, reverting that commit will fix the issue. I will review this a bit more and then I will have an update
